README.md

```diff
@@ -7,16 +7,14 @@
 [![Build Status](https://travis-ci.com/ONSdigital/eq-author-app.svg?branch=master)](https://travis-ci.com/ONSdigital/eq-author-app)
 [![codecov](https://codecov.io/gh/ONSdigital/eq-author-app/branch/master/graph/badge.svg)](https://codecov.io/gh/ONSdigital/eq-author-app)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/f9cbc9e5c531436ca73ee0769a766678)](https://www.codacy.com/app/ONSDigital/eq-author-app)
-![Snyk Vulnerabilities for GitHub Repo](https://img.shields.io/snyk/vulnerabilities/github/ONSDigital/eq-author-app.svg)
```

Comment:
Why remove this?

Reply:
Most of the time it doesn't seem to work for me.. Should I keep it anyways?

Reply:
Its moved back now, but I made the fallback message less verbose.